checkov/github/dal.py

```diff
@@ -54,6 +58,11 @@ def discover(self) -> None:
         self.default_branch_cache = {}
         self.org = os.getenv('GITHUB_ORG', '')
 
+        self.discover_optional_variables()
+
+    def discover_optional_variables(self) -> None:
+        self.org_complementary_metadata['max_admins_count'] = os.getenv('GITHUB_MAX_ADMINS_ORG', 2)
```

Comment:
maybe I missed something, but why do we set this via env var?

Reply:
thought it is a fluid count that can change on how big is the repository. and I am not sure what number to put here.